frontend/src/pages/org/archived-item-detail/archived-item-detail.ts

```diff
@@ -853,43 +853,21 @@ export class ArchivedItemDetail extends BtrixElement {
                 </btrix-desc-list-item>
               `,
         )}
-
+        <btrix-desc-list-item label=${msg("Pages")}>
+          ${this.item
+            ? html`${this.localize.number(this.item.pageCount || 0)}
+              ${pluralOf("pages", this.item.pageCount || 0)}`
+            : html`<sl-skeleton class="h-[16px] w-24"></sl-skeleton>`}
+        </btrix-desc-list-item>
         <btrix-desc-list-item label=${msg("Size")}>
           ${this.item
             ? html`${this.item.fileSize
-                ? html`${this.localize.bytes(this.item.fileSize || 0, {
-                    unitDisplay: "narrow",
-                  })}${this.item.stats?.done
-                    ? html`<span>,</span
-                        ><span
-                          class="tracking-tighter${this.isActive
-                            ? " text-violet-600"
-                            : ""} font-mono"
-                        >
-                          ${this.localize.number(+this.item.stats.done)}
-                          <span class="text-0-400">/</span>
-                          ${this.localize.number(+this.item.stats.found)}
-                        </span>
-                        <span
-                          >${pluralOf("pages", +this.item.stats.found)}</span
-                        >`
```

Comment:
The logic here enabled us to show the current page count (out of total queued and done) for running crawls - might be worth keeping that in? Otherwise it's hard to know when you're deep in a crawl detail for a running crawl how far along that crawl is.

Reply:
Hmm, good catch, this highlights a larger UX issue. I thought this was a leftover from when we migrated towards showing crawl status only in workflows, but we do link to active crawls from the workflow crawl list... I think this was originally an oversight, but it seems like the natural inclination is to see crawl status from the archived item page. Do we want to continue to support this? @Shrinks99 ?

Reply:
Gah, yeah this is a left over that I've been meaning to bring up for a long time!

I would like to remove this to keep the "workflow produces an archived item" concept intact. There isn't much one can do in an archived item that is currently being crawled, and as far as users are concerned an archived item shouldn't exist until a crawl is stopped, canceled, or completes.

Reply:
Created https://github.com/webrecorder/browsertrix/issues/2376 to keep this PR in scope with detail updates